### ISSUE TITLE
Can this be done on Windows 10 Pro for Workstations?

### ISSUE THREAD

**Issue by jdrch** [MicrosoftDocs/windowsserverdocs#2918]
Just wondering the above. Or is it limited to Server SKUs only?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 621d449f-4675-d504-e266-5933283b61e7
* Version Independent ID: c2bd816f-d617-0ac2-7f9f-0f17c06b84bf
* Content: [Mirror-accelerated parity](https://docs.microsoft.com/en-us/windows-server/storage/refs/mirror-accelerated-parity)
* Content Source: [WindowsServerDocs/storage/refs/mirror-accelerated-parity.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/storage/refs/mirror-accelerated-parity.md)
* Product: **windows-server-threshold**
* Technology: **storage-file-systems**
* GitHub Login: @gawatu
* Microsoft Alias: **gawatu**

**Comment by jumpinf00l** [comment]
Requires Storage Spaces D2D which is only available on Server 2016/2019 Data Centre, so no Windows 10 isn’t supported

**Comment by jdrch** [comment]
Thanks so much. It's *really* unclear from Storage Spaces' overall
documentation which features are available in which SKU.

I request that a table completely describing the above be added somewhere.

**Comment by sebbiep1** [comment]
Yes, I have all of the above working on a Windows 10 Pro Workstation install at my home.   I totally agree it is confusing because the above article refers to "Storage Spaces Direct" which is included in Windows Server 2019 Datacenter, Windows Server 2016 Datacenter and is more geared towards clustered servers and storage nodes etc.  Also Microsoft removed ReFS capability from Windows 10 Pro in the Windows 10 Fall Creators Update which was annoying for me as I was already using it.

But the set-up does work on the "standard" Storage Spaces, but via PowerShell rather a GUI - which you would need to use anyway for optimising any Storage Space.   

To be give a precise example I use it as a second level backup for a pair of large Synology NAS's at home.  The ReFS volume is mirror accelerated (8 SSDs with 2 columns) and parity (32 HDDS with 8 columns).   It performs exactly as mentioned in the article above.

This gives me a cumulative 87% storage space efficiency (I don't need more failure resilience as I have at least 2 more resilient copies of everything elsewhere).   Sustained read speed at 550 MB/s is well over my 4Gb/s (using SMB multi-channel) network link to my NAS and write (backup) speed is around 200MB/s which is fine for my use.  It's a bit speed limited because I am using redundant 10 year old consumer HDDs from previous home storage in 4 cheap 8-bay USB enclosures (8 HDDs per USB cable is a bottleneck) - USB is most definitely not a recommended option for large scale Storage Spaces.  However the SSDs are all on LSI HBAs and a 12 core Xeon with 128GB RAM means there is no issue with the parity calculations etc.  It's been very reliable and is very cheap way of getting 100TB of usable backup space.   

I was initially going to use a NTFS tiered virtual disk with both the performance and capacity tiers as mirrors, but Mirror-accelerated parity is far more space efficient with still quite good performance even for bulk writes like backups.   It would be even better for more general usage such as database applications etc due to the way ReFS manages the hot and cold data.

The GUIs in Windows 10 pro (very basic) and Windows Server (less basic) are quite limited.  However with PowerShell, there is a vast amount of options to really tweak and optimise the storage.  However documentation from Microsoft about this is bit thin.   There is slightly better information from storage vendors such as HP and Dell etc and then random blog and forum posts - a few of which contain completely incorrect information.    

The article above only gives the final cmdlet for creating the new volume.   There are plenty of blog posts online about creating a "normal" simple, mirror or parity storage space, but few if any on the full steps to create a mirror-accelerated parity space.   It is a little different because in this case ReFS is taking over some of the heavy lifting from Storage Spaces (it's actually simpler and quicker).   If you intend to try this on Windows 10 Pro Workstation I'd be happy to post all the cmdlets that I used, including quite a few optional extra ones I used for checking all of the stacked object levels that make up a volume and optimising etc.     
It takes 1 to 2 minutes to create a 100TB tiered from scratch.  It's so fast that one evening I created (and deleted) around 140 virtual disks, systematically changing parameters e.g. using every column option from 1 to 32 for both mirror and parity and also adjusting the interleave (stripe) size, block size, Write-Back-Cache size etc until I found (via graphing the disk performance) the ideal set-up for my hardware and usage case. 
Please let me know if you need more information.

**Comment by gmartico** [comment]
Please post the full steps you too. I am in the same boat you were. Trying to get mirror-accelerated parity volume on Windows 10 Pro for Workstations.
